app/controllers/cinema_halls_controller.rb

```diff
@@ -36,7 +37,7 @@ def update
   end
 
   def destroy
-    @cinema_hall = CinemaHall.find(params[:id])
+    @cinema_hall = CinemaHalls::UseCases::Delete.new(id: params[:id]).call
```

Comment:
You don't need to assign the result of use_case to `@cinema_hall`

Reply:
@baniak66, I just want to confirm before commit
You mean that I can change this line into:  `@cinema_hall = CinemaHalls::UseCases::Delete.new.call`  ?

Reply:
When you deleting objects, you will probably don't need result of this action, so you don't need to assign result of this class to `@cinema_hall`. We meant that you should change this line into just:
`CinemaHalls::UseCases::Delete.new(id: params[:id]).call`